app/models/application.rb

```diff
@@ -0,0 +1,171 @@
+# == Schema Information
+#
+# Table name: applications
+#
+#  id                     :integer          not null, primary key
+#  user_id                :integer
+#  track                  :string
+#  over_18                :boolean
+#  previous_attendance    :string
+#  programming_experience :text
+#  reason                 :text
+#  tshirt_size            :string
+#  admin_status           :string
+#  user_status            :string
+#  gender                 :string
+#  dietary_requirements   :json
+#  extra_info             :text
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+
+class Application < ActiveRecord::Base
+  validates :track, :previous_attendance, :programming_experience,
+            :reason, :tshirt_size, :gender, presence: true
+  validates_inclusion_of :over_18, in: [true, false]
```

Comment:
I'd forgotten why this was needed, but after looking around a bit it's the boolean equivalent of `presence: true`. Since it's a boolean, and `false` is an acceptable value the presence test would fail on a false value. See http://stackoverflow.com/questions/3608076/rails-how-do-i-validate-that-something-is-a-boolean